nbbuild/rat-exclusions.txt

```diff
@@ -36,6 +36,7 @@ nbbuild/user.build.properties
 **/Makefile-vc
 **/Makefile-vc-wo-crt
 **/package-list
+groovy.support/src/META-INF.debugger*/**
```

Comment:
This is IMHO not needed. If I'm not mistaken the code, that reads the files is `api.debugger/src/org/netbeans/api/debugger/Lookup.java` and there comment sign `#` is explicitly mentioned, so I would add the ALv2 header directly. There are also debugger related files `spi.debugger.ui/build/classes/META-INF/debugger/ThreadsView/org.netbeans.spi.viewmodel.ColumnModel` that show the header.

So I would not add it to the exlusion list, but add the header.

Reply:
I'll add the license header later. Thanks!